Lib/http/server.py

```diff
@@ -149,6 +151,42 @@ class ThreadingHTTPServer(socketserver.ThreadingMixIn, HTTPServer):
     daemon_threads = True
 
 
+class HTTPSServer(HTTPServer):
+    def __init__(self, server_address, RequestHandlerClass,
```

Comment:
Can you wrap all lines under 80 chars in this function? I think the ALPN protocols line exceeds 80 chars and raising RuntimeError as well.

Reply:
done :+1: